nilearn/_utils/ndimage.py

```diff
@@ -38,6 +47,40 @@ def largest_connected_component(volume):
     return labels == label_count.argmax()
 
 
+def largest_connected_component_img(imgs):
+    """ Return the largest connected component of an image or list of images.
+
+    Parameters
+    ----------
+    imgs: Niimg-like object or iterable of Niimg-like objects
+        See http://nilearn.github.io/manipulating_images/input_output.html
+        Image(s) to extract the largest connected component from.
```

Comment:
I had a short discussion with @KamalakerDadi  about this. We were a bit unsure about how ndimage would behave in the case of 4D images. I am going to look into this tonight.